packages/my-queries-storage/src/provider.ts

```diff
@@ -0,0 +1,50 @@
+import { createContext, useContext } from 'react';
+import type { QueryStorageOptions } from './compass-query-storage';
+import type { PipelineStorage } from './pipeline-storage';
+import type { FavoriteQueryStorage, RecentQueryStorage } from './query-storage';
```

Comment:
These are not implementations but the interfaces. I renamed the implementation to be prefixed with `Compass` and moved them to separate files also prefixed with `compass-`.